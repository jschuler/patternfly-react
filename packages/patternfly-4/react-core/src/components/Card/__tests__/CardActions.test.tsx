// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { CardActions } from '../CardActions';

describe('CardActions test', () => {
  it('CardActions should match snapshot', () => {
    const view = shallow(
      <CardActions
        children={<>ReactNode</>}
				className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
