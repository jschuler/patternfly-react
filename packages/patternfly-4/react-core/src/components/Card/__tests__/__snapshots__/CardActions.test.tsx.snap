// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardActions test CardActions should match snapshot 1`] = `
<div
  className="pf-c-card__actions ''"
>
  ReactNode
</div>
`;
